assemblies/solr-proxy-service/src/main/feature/feature.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
+
+    <feature name='Solr Proxy Service' description='Solr Proxy Service' version='${project.version}'>
+        <bundle>mvn:org.pentaho.ctools/solr-proxy/${project.version}</bundle>
```

Comment:
The bundle should be added as dependency in the pom file and inserted in the feature by the karaf-maven-plugin plugin. But we probably don't need the feature anyways, the bundle is enough.